src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -105,12 +105,32 @@ internal sealed class LogLossFactory : ISupportSdcaClassificationLossFactory, IS
         IClassificationLoss IComponentFactory<IClassificationLoss>.CreateComponent(IHostEnvironment env) => new LogLoss();
     }
 
+    /// <summary>
+    /// The Log Loss, also known as the Cross Entropy Loss. It is commonly used in classification tasks.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// The Log Loss function is defined as:
+    ///
+    /// $L(f(\vec{x}), y) = -y ln(f(\vec{x})) - (1 - y) ln(1 - f(\vec{x}))$
+    ///
+    /// where $f(\vec{x})$ is the predicted probability of belonging to the positive class and $y \in \{0, 1\}$ is the true label.
```

Comment:
>f(\vec{x}) [](start = 15, length = 10)

Would it be better to give this a name? Maybe \hat(y)?

Reply:
And I would also explain that it is the sigmoid of the score.

---
In reply to: [278258057](https://github.com/dotnet/machinelearning/pull/3560#discussion_r278258057) [](ancestors = 278258057)